apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html

```diff
@@ -11,24 +11,18 @@
 </fd-segmented-button>
 
 <div style="padding: 1rem; overflow: auto">
-    <fd-fixed-card-layout (cardDraggedDropped)="cardDraggedDropped($event)">
+    <fd-fixed-card-layout (cardDraggedDropped)="cardDraggedDropped($event)" [disableDragDrop]="true">
```

Comment:
Does it still need `(cardDraggedDropped)="cardDraggedDropped($event)"` if `[disableDragDrop]="true"`?